### PR TITLE
Add disable annotate env var

### PR DESCRIPTION
**Pull request by mafernando**
Disable annotate on migrations by default and check for ENABLE_ANNOTATIONS if in development

### CODE REVIEW

**Comment by mafernando** [comment]
@stackus - i think this is what you had in mind?

**Comment by stackus** [comment]
Could it be made to be transparent for the developers? As it is these changes will require devs who modify the schema to remember to enable this and to also run the rake command up update the annotations.

**Comment by mafernando** [comment]
Sure. Annotate now does not run by default if in "development" unless 'DISABLE_ANNOTATIONS = true' is an ENV variable

File: lib/tasks/auto_annotate_models.rake [hunk 1]
```diff
@@ -30,5 +30,5 @@ if Rails.env.development?
     )
   end
 
-  Annotate.load_tasks
+  Annotate.load_tasks unless ENV.key? 'DISABLE_ANNOTATIONS'
```
Comment (stackus) [inline lib/tasks/auto_annotate_models.rake]: :+1: 

Also yes `key?` vs the deprecated `has_key?`.

**Pull request merged**